frontend/src/models/dashboardItemsModel.tsx

```diff
@@ -34,6 +45,15 @@ export const dashboardItemsModel = kea<dashboardItemsModelType<DashboardItemType
                 },
             })
         },
+        updateDashboardItem: async ({ id, ...payload }) => {
+            if (!Object.entries(payload).length) {
+                return
+            }
+            const response = await api.update(`api/dashboard_item/${id}`, payload)
+            actions.setDashboardItemMode(null)
+            insightLogic.actions.setDashboardItem(response)
+            return response
```

Comment:
Q: is the return needed here? Why?

Reply:
nope! Thought I would need it but I didn't 😃 I shall remove it